index.html

```diff
@@ -4,6 +4,6 @@
         <title>Example</title>
     </head>
     <body>
-        <h1>Welcome to Potionfy.</h1>
+        <h1>Welcome to Potionfy!!!!</h1>
```

Comment:
Can you add 'Enjoy your stay! and make it red?

Reply:
sure !